PostHog/Replay/PostHogReplayIntegration.swift

```diff
@@ -170,6 +171,14 @@
             if postHog.config.sessionReplayConfig.captureNetworkTelemetry {
                 sessionSwizzler?.swizzle()
             }
+
+            // start captures console logs
+            if postHog.config.sessionReplayConfig.captureLogs {
+                consoleLogInterceptor = PostHogConsoleLogInterceptor()
+                consoleLogInterceptor?.startCapturing(config: postHog.config) { [weak self] output in
```

Comment:
we should not allow multiple instances at the same time
https://github.com/PostHog/posthog-android/blob/d232586ce6b5043d716f108d06316d8de4714c31/posthog-android/src/main/java/com/posthog/android/replay/internal/PostHogLogCatIntegration.kt#L27

Reply:
ahh yeah cool! thx